src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -7,7 +7,7 @@ const defaultArgs = {
   label: 'Checkbox',
 };
 
-export default {
+const meta: Meta<typeof Checkbox> = {
```

Comment:
In cases like this, there were type errors that prevented tests from passing. Following the docs from storybook, this is the recommended typing to apply. What's weird is that types didn't break down on ALL tests, so not sure when types would get lost yet.

It's likely we should make these consistent in one way or the other, and favor aligning with this pattern since it mirrors [the docs](https://storybook.js.org/docs/react/writing-stories/introduction#default-export)

Reply:
> it's likely we should make these consistent in one way or the other, and favor aligning with this pattern since it mirrors the docs

:+1:

I did something similiar when upgrading the Learning platform to Storybook v7. I went though and changed every `as Meta<Args>` to `satisfies Meta<Args>` which is a bit better than `as`. Not saying `satifies` is better than `const meta: Meta <...>` just pointing out I ran into similar weirdness

Reply:
Yep yep, i recall that one, and tested it on a few places. I think there may be more issues with the story files since that didn't have an effect in some cases. It's a little bit mysterious...